Games/baj.2m.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# <bitbar.title>BAJ</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Justin Honold</bitbar.author>
+# <bitbar.author.github>astrostl</bitbar.author.github>
+# <bitbar.desc>Checks for active turns at http://boiteajeux.net every two minutes</bitbar.desc>
+# <bitbar.image></bitbar.image>
```

Comment:
I upload an image to a new issue and use the URL it returns. The image link is still good even without submitting the issue.

![dorzlbuq1j](https://cloud.githubusercontent.com/assets/13259/12343655/5cc17114-bafb-11e5-8c7c-ac3240f0afd2.gif)